nomenclature/validation_schemas/region_mapping_schema.yaml

```
@@ -0,0 +1,52 @@
+$schema: 'https://json-schema.org/draft/2020-12/schema'
+title: RegionAggregationMapping
+description: >
+  This schema is used to do the first stage of validation for yaml region
+  mapping files.
```

Shortening to one line (below 88 characters)
```suggestion
  This schema is used for the first stage of validation for yaml region mapping files.
```